dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -13,6 +13,8 @@
 //= require jquery-ui
 //= require video
 //= require bootstrap
+//= require react
```

Comment:
Will this conflict with https://github.com/code-dot-org/code-dot-org/blob/22dc58cb11aa4948bac10b18311019be9ce4c943/dashboard/config/application.rb#L95?